### ISSUE TITLE
Merge into core

### ISSUE THREAD

**Issue by saulshanabrook** [jupyterlab/jupyterlab-statusbar#106]
We should merge this extension into core so that we can easier add things to it when we add new features to core. See https://github.com/jupyterlab/jupyterlab/issues/5352 for the issue there.

@ian-r-rose I see you have been assigned to that issue. Is there anything you want from the developers here to get this started?

**Comment by ian-r-rose** [comment]
No, I'm going to wait for the dust to settle a bit with 0.35 and then do it.

**Comment by saulshanabrook** [comment]
Sounds good.
